### PR TITLE
Add packages config option to spark task.

### PR DESCRIPTION
**Pull request by blooshirt**
Add packages configuration option to spark task in order to support applications like:
https://github.com/databricks/spark-csv#using-with-spark-shell

### CODE REVIEW

File: luigi/contrib/spark.py [hunk 1]
```diff
@@ -122,6 +122,10 @@ def jars(self):
         return self._list_config(configuration.get_config().get("spark", "jars", None))
 
     @property
+    def packages(self):
```
Comment (erikbern) [inline luigi/contrib/spark.py]: can we make this cleaner using a `Config` class instead?

**Comment by erikbern** [comment]
LGTM

File: luigi/contrib/spark.py [hunk 2]
```diff
@@ -122,6 +122,10 @@ def jars(self):
...
+        return self._list_config(configuration.get_config().get("spark", "packages", None))
```
Comment (Tarrasch) [inline luigi/contrib/spark.py]: Yea use `Config` class for this please.

Comment (blooshirt) [inline luigi/contrib/spark.py]: Are you suggesting I make a new class just for the new parameter?

Comment (erikbern) [inline luigi/contrib/spark.py]: Let's just merge this for now and deal with the config refactoring in a
separate PR

Comment (Tarrasch) [inline luigi/contrib/spark.py]: Yes, see http://luigi.readthedocs.org/en/latest/api/luigi.configuration.html. As an advantage, you get command line parameters for free.

Comment (erikbern) [inline luigi/contrib/spark.py]: yeah but this complexity has been there for a long time in the spark class, so we should fix it in a separate PR

Comment (blooshirt) [inline luigi/contrib/spark.py]: Agreed Erik. I'll start working on a separate PR for refactoring the entire config of the spark class. In the mean-time this functionality is necessary so I think we should just merge this in.

Comment (Tarrasch) [inline luigi/contrib/spark.py]: Ah I thought it was the first use of `get_config()`. Great then. :)

**Pull request merged**